geographies-api/docker-compose-dev.yml

```diff
@@ -0,0 +1,49 @@
+services:
+  app:
+    container_name: geographies-api-app
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    ports:
+      - 8080:8080 # @related: PORT_NUMBER
+    environment:
+      AWS_ACCESS_KEY_ID: test
+      AWS_SECRET_ACCESS_KEY: test
+      AWS_ENDPOINT_URL: http://localstack:4566
+      AWS_DEFAULT_REGION: eu-west-1
+      GEOGRAPHIES_BUCKET: test-bucket
+      CDN_URL: http://localstack:4566/test-bucket
+    volumes:
+      - .:/app
+    working_dir: /app
+    command: |
+      sh -c '
+        fastapi dev app/main.py --port 8080 --host 0.0.0.0 &
+        until curl -s http://localhost:8080/geographies/docs > /dev/null; do sleep 2; done;
+        until aws --endpoint-url=http://localstack:4566 s3 ls s3://test-bucket; do
+          echo "Waiting for geographies bucket to exist..."
+          sleep 2
+        done
+        curl -X GET http://localhost:8080/geographies/populate-s3-bucket
+        wait
```

Comment:
Interesting way to do this! [We use a similar pattern with initial-data](https://github.com/search?q=repo%3Aclimatepolicyradar%2Fnavigator-backend%20initial-data&type=code) in other services.

later: I wonder if we could consolidate into a new pattern?